docs/pages/kcardgrid.vue

```diff
@@ -964,10 +765,12 @@
 
   import useKResponsiveWindow from '../../lib/composables/useKResponsiveWindow';
   import DocsKCard from '../pages-components/DocsKCard';
+  import DocsExample from '../common/DocsExample'; // Corrected the import typo
```

Comment:
Would you remove this import and instead register `DocsExample` in [the `load-common-components.js` file](https://github.com/learningequality/kolibri-design-system/blob/05119fedc4328e370389b52f41749d0904945660/docs/plugins/load-common-components.js)? This will make `DocsExample` available in all documentation pages without the need to import it.